bot/bot.py

```diff
@@ -36,6 +41,16 @@ def __init__(self, redis_session: RedisSession, **kwargs):
         self.redis_session = redis_session
         self.loop.create_task(self.check_channels())
         self.loop.create_task(self.send_log(self.name, "Connected!"))
+        self.loop.create_task(self.process_ext_not_found())
```

Comment:
Is this guaranteed to run after all the extensions have been loaded?

Reply:
> Is this guaranteed to run after all the extensions have been loaded?

It should, yes, and while testing it did.

Reply:
Yeah, the event loop gets ran after the bot instance is created